include/FIFOSampleBuffer.h

```diff
@@ -1,12 +1,12 @@
 ////////////////////////////////////////////////////////////////////////////////
 ///
-/// A buffer class for temporarily storaging sound samples, operates as a 
+/// A buffer class for temporarily storaging sound samples, operates as a

```

Comment:
Can you please keep the changes to a minimum?
It will be a pain in the ass to rebase this stuff.

I don't see a reason to do any cosmetics

Reply:
It's auto formatting. Guess i have to edit this in notepad...

Reply:
Removed all whitespace changes